### ISSUE TITLE
Re-ranking?

### ISSUE THREAD

**Issue by ttdd11** [wangguanan/Pytorch-Person-REID-Baseline-Bag-of-Tricks#10]
I read this: Note that our implementation uses no the center loss and re-ranking

Does this mean that this isn't using re-ranking. Also, do you have a pre-trained model I can test with?

**Comment by wangguanan** [comment]
Yes, we didn't use re-ranking.
We will upload a pre-trained model later.
